common/common.h

```diff
@@ -168,6 +168,8 @@ char *mp_tprintf_buf(char *buf, size_t buf_size, const char *format, ...)
 
 char **mp_dup_str_array(void *tctx, char **s);
 
+#define mp_str_len(s) (sizeof("" s) - 1)
```

Comment:
```suggestion
// returns length of static constant
#define mp_str_len(s) (sizeof("" s) - 1)
```

but actually this isn't really needed for the code in `osd_libass.c`, compilers will constant fold `strlen` these days

Reply:
This is not guaranteed by standard and will not work with fno-builtin, but wither way it doesn't really matter.